tumblebit/__init__.py

```diff
@@ -24,12 +25,18 @@
 ########################################################
 
 # Change path to where libressl library is
-# TODO: Add an option to specify the path in some sort of
-#       config file.
-if(system() == "Darwin"):
-    path = "/usr/local/opt/libressl/lib/libssl.dylib"
+try:
+    config.LIBRESSL_PATH
```

Comment:
Oh, I see what you're saying. set the path in the try statement instead of in else madness
`